### ISSUE TITLE
Get `Error while waiting for Protractor to sync with the page: true` with Angular2

### ISSUE THREAD

**Issue by romanovma** [Codeception/CodeceptJS#372]
My app navigates to the overview page after user logged in so I test it like this:
```
    I.amOnPage('/users/roles');
    I.click('role1', 'myComponent');
    I.seeInCurrentUrl('/users/overview');
    I.see('role1', 'nav');
```

but get the error message:

```bash
Uncaught Error: Error while waiting for Protractor to sync with the page: true
      at node_modules/protractor/built/browser.js:272:23
      at ManagedPromise.invokeCallback_ (node_modules/selenium-webdriver/lib/promise.js:1379:14)
      at TaskQueue.execute_ (node_modules/selenium-webdriver/lib/promise.js:2913:14)
      at TaskQueue.executeNext_ (node_modules/selenium-webdriver/lib/promise.js:2896:21)
      at asyncRun (node_modules/selenium-webdriver/lib/promise.js:2775:27)
      at node_modules/selenium-webdriver/lib/promise.js:639:7
      at process._tickCallback (internal/process/next_tick.js:103:7)
```

When I add `amOnPage` directive then everything works fine:
```
    I.amOnPage('/users/roles');
    I.click('role1', 'myComponent');
    I.amOnPage('/users/overview');
    I.seeInCurrentUrl('/users/overview');
    I.see('role1', 'nav');
```

Any ideas how I can test that user is redirected to overview page after he/she is logged in?

### Details

* CodeceptJS version: 0.4.14
* NodeJS Version: 7.2.1
* Operating System: Mac OSX El Capitan
* Protractor: 4.0.14
* Configuration file:

```js
useAllAngular2AppRoots: true,
rootElement: 'app-root'
```

**Comment by romanovma** [comment]
seems that the real cause wasn't the row described above but the fact that I ran several scenarios without restarting the browser(some bug there?). 
I got `sessionId is null` error or `local settings are not available for 'data' urls` all the time I ran more than one scenario. Not sure about actual reason but once I added `restart: true` to the protractor helper in codecept config, everything works smoothly.
